force-app/main/default/chat/lwc/messageInput/messageInput.js

```diff
@@ -0,0 +1,26 @@
+//todo:: const labels in html
+import {LightningElement} from 'lwc';
+
+const SEND_EVENT = 'send';
+const ENTER_KEY = 'Enter';
+
+export default class MessageInput extends LightningElement {
+
+    messageBody = '';
+
+    handleSendMessage() {
+        this.dispatchEvent(new CustomEvent(SEND_EVENT, {'detail': this.messageBody}));
+        this.messageBody = '';
+    }
+
+    handleChangeMessage(event) {
+        this.messageBody = event.target.value;
```

Comment:
I think you could get the entire input right before the sending. There is no necessity to handle each entered symbol.

Reply:
fixed in 8d870610e229cc8a420aa5daeddcdb7010ca08a5